### ISSUE TITLE
Change to defaultLayout can give v3.1.0 ENOENT main.handlebars

### ISSUE THREAD

**Issue by aubergene** [ericf/express-handlebars#252]
Previously we hadn't specified `layout: false` and so it started looking for `main.handlebars` giving a ENOENT error. This was unexpected and probably shouldn't have be a minor version bump.

The change to the defaults was in #249 

Creating this issue just so other people can find it if they encounter the same problem. We fixed it by adding `layout: false` to the handlebars options.

**Comment by amypellegrini** [comment]
I'm having the same issue but setting layout to false is not fixing it. Any ideas?

This issue seems to be a duplicate of #253 where I posted the same question.

**Comment by donmccurdy** [comment]
Agreed that per https://semver.org/, this should have been a major version bump. Thanks for creating the issue to track this.

@amypellegrini using `layout:false` did fix the issue for me. From #253 it looks like your issue was solved?

**Comment by amypellegrini** [comment]
@donmccurdy Yes, I've fix the issue by setting `layout`to false. My previous comment refers to some difficulty I encountered trying to implement the fix, as it wasn't evident for me how this setting was used.
